networkmonitor/src/main/java/ca/rmen/android/networkmonitor/provider/NetMonDatabase.java

```diff
@@ -82,6 +82,7 @@
             + NetMonColumns.DEVICE_LATITUDE + " REAL, "
             + NetMonColumns.DEVICE_LONGITUDE + " REAL, "
             + NetMonColumns.DEVICE_POSITION_ACCURACY + " REAL, "
+            + NetMonColumns.DEVICE_SPEED + " REAL, "
```

Comment:
Missing the upgrade scenario.  See, for example, SQL_UPDATE_TABLE_NETWORKMONITOR_V15_SERVICE_STATE.

Need to update the DATABASE_VERSION to 16.